### DISCUSSION TITLE
R.E.S.P.E.C.T.

### DISCUSSION THREAD

**Discussion by AndreaDowning** [TheLightCollective/PatientLedAI#6] [Ideas]
We might consider re-organizing the content of our rights into **R.E.S.P.E.C.T**:   

These are the current sections: 

1. Patient-Led Governance
2. Independent Duty to Patients
3. Transparency
4. Self-Determination
5. Identity Security and Privacy
6. Right of Action
7. Shared Benefit

Here is how each section would map:  Right of Action, Equal Benefit, Self Determination, Patient-led Governance, Equal & Independent Duty to Patients, Transparency.  

1.  Right of Action (formerly # 6)
2. Equal Benefit  (formerly 7 Shared benefit)
3. Self-Determination (formerly 4)
4. Patient-Led Governance (Formerly 1)
5. Equal & Independent Duty to Patients (Formerly 2 Independent Duty to Patients)
6. Collective Consent / Control ( Phrasing for Identity Security & Privacy)
7. Transparency (Formerly 3)

**Comment by hugooc** [comment]
Excellent idea. Well done.

**Reply by AndreaDowning** [reply]
Thank you Hugo!!  the only weak spot is the Collective Consent /Control part if you have any ideas here.

Another thing we're thinking more about is how we might go further to develop measures and test these 7 things in a real clinical setting.  For example I am looking into [this opportunity further.](https://www.pcori.org/funding-opportunities/announcement/advancing-science-engagement-research-pcori-funding-announcement-cycle-2-2024)
